apidoc/Titanium/UI/Tab.yml

```diff
@@ -224,7 +224,8 @@ properties:
     summary: The icon inset or outset for each edge.
     description: | 
         Use this property for example to center an icon without providing a title. To do that, set the
-        insets to `{top:6, bottom:-6, right:0, left:0}`.
+        insets to `{top:6, left: 0}`. The `right` and `bottom` are calculated internally to prevent the
```

Comment:
I'd leave out `left:0` since that's the default (and you would mostly use `top` anyway)